homeassistant/components/edl21/__init__.py

```diff
@@ -1 +1,18 @@
 """The edl21 component."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+PLATFORMS = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Set up EDL21 integration from a config entry."""
+    await hass.config_entries.async_forward_entry_setups(config_entry, PLATFORMS)
+    return True
+
```

Comment:
The suggested change in async_setup_entry
```
return hass.config_entries.async_forward_entry_setups(config_entry, PLATFORMS)
```
results in an error when starting HomeAssistant
```
2023-02-24 19:39:15.198 ERROR (MainThread) [homeassistant.config_entries] edl21.async_setup_entry did not return boolean
/workspaces/home-assistant/homeassistant/config_entries.py:389: RuntimeWarning: coroutine 'ConfigEntries.async_forward_entry_setups' was never awaited
Coroutine created at (most recent call last)
  File "/home/vscode/.vscode-server/extensions/ms-python.python-2023.2.0/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 124, in _run_code
    exec(code, run_globals)
  File "/workspaces/home-assistant/homeassistant/__main__.py", line 226, in <module>
    sys.exit(main())
  File "/workspaces/home-assistant/homeassistant/__main__.py", line 214, in main
    exit_code = runner.run(runtime_conf)
  File "/workspaces/home-assistant/homeassistant/runner.py", line 179, in run
    return loop.run_until_complete(setup_and_run_hass(runtime_config))
  File "/usr/local/lib/python3.10/asyncio/base_events.py", line 636, in run_until_complete
    self.run_forever()
  File "/usr/local/lib/python3.10/asyncio/base_events.py", line 603, in run_forever
    self._run_once()
  File "/usr/local/lib/python3.10/asyncio/base_events.py", line 1898, in _run_once
    handle._run()
  File "/usr/local/lib/python3.10/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/workspaces/home-assistant/homeassistant/config_entries.py", line 383, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/workspaces/home-assistant/homeassistant/components/edl21/__init__.py", line 12, in async_setup_entry
    return hass.config_entries.async_forward_entry_setups(config_entry, PLATFORMS)
  result = False
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
```
I could not find out why, so I left it as it was (for now).